.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -0,0 +1,285 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
```

Comment:
@jatakiajanvi12, did we test all the scenarios? if not, can we please test all the scenarios? if you have tested all the scenarios, can you please add to the PR description with list of scenarios being validated.

Reply:
@ganga1980 I have updated the description with a list. Let me know if you think I missed anything.